src/main/webapp/i18n/en/result.json

```diff
@@ -28,7 +28,7 @@
             "hasFeedback": "Has Feedback",
             "assessmentType": "Assessment Type",
             "assessor": "Reviewer",
-            "feedbacks": "Feedbacks",
+            "feedbacks": "Feedback",
```

Comment:
Variable name is inconsistent with it's content.

Reply:
This is the case with quite a lot of changes; I felt it was in the interest of keeping the PR cohesive to ignore that at first and then fix it later. Would be willing to author that subsequent PR.